spec/order_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'order'
+
+describe Order do
+  let(:order) { Order.new }
+  it 'is an empty list by default' do
+    expect(subject.items).to be_empty
+  end
+
+  it 'can add items to an order' do
```

Comment:
I appreciate there's only 2 tests in this spec, but potentially using `describe '#order' do; end` block may give some clarity when you need to add more tests in